### DISCUSSION TITLE
REVTex template

### DISCUSSION THREAD

**Discussion by killiansheriff** [typst/typst#1645] [Ideas]
Any plans for creating a template similar to REVTex (https://journals.aps.org/revtex)? 

Thanks!

**Comment by Qottmann** [comment]
This would be awesome to have, received some demand from physics colleagues with overleaf being down today 😁

**Comment by iqsnider** [comment]
Anyone gotta status update here? I start my phd in 3 weeks and I'm not mentally ready to return to the trauma of conflicting latex packages and egregious compile times

**Comment by WeileiZeng** [comment]
check out this one    https://github.com/skilledwolf/revtypst
